low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -32,18 +39,133 @@ class ServerMap(ResourceMap):
 
     def __init__(self):
         """Initialize server"""
-        pass
+        super().__init__()
+        self.sysfs = ToolFactory().get_instance('sysfs')
+        self.sysfs.initialize()
+        self.sysfs_base_path = self.sysfs.get_sysfs_base_path()
+        self.cpu_path = self.sysfs_base_path + "devices/system/cpu/"
+
+        self.server_frus = {
+            'cpu': self.get_cpu_info
+        }
 
     def get_health_info(self, rpath):
         """
         Get health information of fru in given resource id
 
         rpath: Resource id (Example: nodes[0]>compute[0]>hw>disks)
         """
-        pass
+        info = {}
```

Comment:
why line 58-128 is showing up in this PR, if you have rebased against 'health' feature branch ?

Reply:
Mariyappan's PR had updated functions for storage but not for server.
https://github.com/Seagate/cortx-monitor/blob/health/low-level/files/opt/seagate/sspl/setup/resource_map/server.py

Reply:
Okay